### PR TITLE
sys-auth/nss-pam-ldapd: fix musl build

### PR DESCRIPTION
**Pull request by cfuga**
Signed-off-by: Cristian Othón Martínez Vera <cfuga@cfuga.mx>
Bug: https://bugs.gentoo.org/713612
Bug: https://bugs.gentoo.org/716272

* add sys-libs/musl-nscd as dependency for musl, to fix ```unknown type name nss_status_t``` error
* patch for missing NETDB_INTERNAL

### CODE REVIEW

File: sys-auth/nss-pam-ldapd/files/nss-pam-ldapd-0.9.12-netdb-defines.patch [hunk 1]
```diff
@@ -0,0 +1,26 @@
+diff -uNr a/nss/hosts.c b/nss/hosts.c
```
Comment (thesamesam) [inline sys-auth/nss-pam-ldapd/files/nss-pam-ldapd-0.9.12-netdb-defines.patch:1]: Please add bug references to any relevant Gentoo bugs to the top of the patch. Also could you run `scrub-patch` from `iwdevtools` on it?

**Comment by thesamesam** [comment]
Thanks! I'm going to restore all the keywords but arm/ppc/ppc64 then file a rekeywording bug for those.

**Comment by cfuga** [comment]
I noticed the new ebuild is already in the master tree. Should I close this PR?

**Comment by thesamesam** [comment]
Yes - sorry! When I squashed, I lost the closes tag.

Merged in 090dc863c60b149b5942951fb2326472c6e70281. Thank you!

**Pull request closed without merging**